pkg/flare/archive.go

```diff
@@ -368,6 +376,45 @@ func zipHealth(tempDir, hostname string) error {
 	return err
 }
 
+func zipStackTraces(tempDir, hostname string) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
+	pprofURL := fmt.Sprintf("http://localhost:%s/debug/pprof/goroutine?debug=2",
+		config.Datadog.GetString("expvar_port"))
+
+	defer cancel()
+
+	client := http.Client{}
+	req, err := http.NewRequest(http.MethodGet, pprofURL, nil)
+	if err != nil {
+		return err
+	}
+
+	resp, err := client.Do(req.WithContext(ctx))
+	if err != nil {
+		return err
+	}
+
```

Comment:
The impact must be very limited but I think you should `defer resp.Body.Close()` here!

Reply:
Definitely! Nice catch.